rpc-jobs/multi-node-aio.yml

```diff
@@ -1,13 +1,26 @@
 - job:
-    name: 'OnMetal_Multi_Node_AIO_Prepare_Deployment'
+    name: 'OnMetal_Multi_Node_AIO'
     project-type: workflow
     build-discarder:
         days-to-keep: 30
     concurrent: true
     parameters:
       - string:
-          name: NODE
-          description: "Name or label of Jenkins node to run on"
+          name: REGION
```

Comment:
These are in single_use_slave_params in rpc-gating/rpc-jobs/params.yml

Maybe that could have a better name, cloud_slave_params?